shuttle/views.py

```diff
@@ -9,6 +9,10 @@ class View(FlaskView):
     def index(self):
         return render_template('index.html')
 
+    @route('/shuttle-stats')
+    def stats(self):
+        return render_template('shuttle_stats.html')
+
```

Comment:
Could you move this route inside of a new init file and put it in a new directory called `schedules`. The path would be /shuttle/schedules/\_\_init\_\_.py

Reply:
Set it up similarly to this file https://github.com/betheluniversity/writing-center-python/blob/master/writing_center/appointments/__init__.py 

Basically you need to create a class called `SchedulesView` and then put the route in it.

Reply:
You would then have to register the view in the base level init. Check in the current base level init to see how View is imported and registered.

Reply:
Sounds good. I just made those changes